testFixtures/sift/server/node2/node2.go

```diff
@@ -2,7 +2,7 @@ package node2
 
 import (
 	"fmt"
-	"server/utils"
+	"github.com/redsift/sandbox-go/testFixtures/sift/server/utils"
```

Comment:
This is going to pulled into our sandbox, so package name should stay as "server/utils", otherwise we'll have an issue

Reply:
relative paths don't work anymore
see https://github.com/golang/go/issues/26645#issuecomment-408572701